http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/NettyConfiguration.java

```diff
@@ -53,10 +53,8 @@ public int connectionTimeout() {
     /**
      * @see NettySdkHttpClientFactory.Builder#maxConnectionsPerEndpoint(Integer)
      */
-    @ReviewBeforeRelease("Does it make sense to use this value? Netty's implementation is max connections" +
-                         " per endpoint so if it's a shared client it doesn't mean quite the same thing.")
     public int maxConnectionsPerEndpoint() {
-        return factory.maxConnectionsPerEndpoint().orElseGet(() -> serviceDefaults.get(MAX_CONNECTIONS));
```

Comment:
Do we not use `maxConnectionsPerEndpoint` anymore?

Reply:
It gets merged into the service defaults map so we don't need to check it in the factory.